packages/gitbook/src/lib/api.ts

```diff
@@ -724,126 +727,34 @@ export const getSite = cache({
 });
 
 /**
- * List all the site-spaces variants published in a site.
+ * List the scripts to load for the site.
  */
-export const getSiteSpaces = cache({
-    name: 'api.getSiteSpaces',
-    tag: ({ siteId }) => getAPICacheTag({ tag: 'site', site: siteId }),
-    get: async (
-        args: {
-            organizationId: string;
-            siteId: string;
-            /** Site share key that can be used as context to resolve site space published urls */
-            siteShareKey: string | undefined;
-        },
-        options: CacheFunctionOptions,
-    ) => {
-        const response = await getAll((params) =>
-            api().orgs.listSiteSpaces(
-                args.organizationId,
-                args.siteId,
-                {
-                    ...params,
-                    ...(args.siteShareKey ? { shareKey: args.siteShareKey } : {}),
-                },
-                {
-                    ...noCacheFetchOptions,
-                    signal: options.signal,
-                },
-            ),
-        );
-
-        return cacheResponse(response, {
-            revalidateBefore: 60 * 60,
-            data: response.data.items.map((siteSpace) => siteSpace),
-        });
-    },
-});
-
-export const getSiteStructure = cache({
-    name: 'api.getSiteStructure',
-    tag: ({ siteId }) => getAPICacheTag({ tag: 'site', site: siteId }),
+export const getPublishedContentSite = cache({
+    name: 'api.getPublishedContentSite',
+    tag: (organizationId, siteId) => getAPICacheTag({ tag: 'site', site: siteId }),
     get: async (
-        args: {
-            organizationId: string;
-            siteId: string;
-            /** Site share key that can be used as context to resolve site space published urls */
-            siteShareKey: string | undefined;
-        },
+        organizationId: string,
+        siteId: string /** Site share key that can be used as context to resolve site space published urls */,
+        siteShareKey: string | undefined,
```

Comment:
Can we use a named object instead? like before

a list of argument is error prone